programs/lib/linker/flexpret.ld

```diff
@@ -21,12 +21,18 @@
 /* 
  * Taken from https://github.com/apache/nuttx/blob/master/boards/risc-v/c906/smartl-c906/scripts/ld.script
  */
-INCLUDE flexpret_config.ld
+INCLUDE flexpret_hwconfig.ld
+INCLUDE flexpret_swconfig.ld
```

Comment:
These files are generated by scala (hwconfig) and make (swconfig). swconfig only contains stacksize, but could contain more in the future.

Reply:
And hwconfig contains the other variables: DSPM_END, DSPM_START, ...